### ISSUE TITLE
i get error sometimes

### ISSUE THREAD

**Issue by godlessturtle** [mobiten/flutter_staggered_animations#1]
it works well, but i get some error sometimes
```
[VERBOSE-2:ui_dart_state.cc(148)] Unhandled Exception: 'package:flutter/src/animation/animation_controller.dart': Failed assertion: line 451 pos 7: '_ticker != null': AnimationController.forward() called after AnimationController.dispose()
AnimationController methods should not be used after calling dispose.
#0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:40:39)
#1      _AssertionError._throwNew (dart:core-patch/errors_patch.dart:36:5)
#2      AnimationController.forward (package:flutter/src/animation/animation_controller.dart:451:7)
#3      _AnimationExecutorState.initState.<anonymous closure> (package:flutter_staggered_animations/src/animation_executor.dart:36:63)
#4      new Future.delayed.<anonymous closure> (dart:async/future.dart:316:39)
#5      _rootRun (dart:async/zone.dart:1120:38)
#6      _CustomZone.run (dart:async/zone.dart:1021:19)
#7      _CustomZone.runGuarded (dart:async/zone.dart:923:7)
#8      _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zon<…>
```

**Comment by MrLepage** [comment]
Thx for pointing that out.
Will be fixed in **0.1.2**

**Comment by rizalkotlin** [comment]
I have the same issue and I will waiting be fixed in 0.1.2, thank you

**Comment by MrLepage** [comment]
@godlessturtle @rizalkotlin 
**0.1.2** has been released and fixed this issue.
You can safely update, there is no breaking change.
